### ISSUE TITLE
AppleIntelSKLGraphicsFamebuffer

### ISSUE THREAD

**Issue by Nxys** [sakoula/XiaoMi-Air-6200U#1]
XiaoMi Air 13.3 Skylake-U 2016
Mac OS Mojave 10.14
I don't know where the problem is. ⊙﹏⊙
This is my efi:
[EFI.zip](https://github.com/sakoula/XiaoMi-Air-6200U/files/3135628/EFI.zip)

**Comment by sakoula** [comment]
What is the issue that you are having? Are you using the exact versions of the kexts and clover that I describe on the guide? I have not upgraded yet on 10.14.4. I am still on 10.14.3

There is a typo on the guide on [this section](https://github.com/sakoula/XiaoMi-Air-6200U#move-kexts-to-libraryextentions)

should be like:

### Move kexts to `/Library/Extensions`
[up up up](#)

The right way to load kexts is **not** by injecting them through clover but installing them in `/Library/Extensions` and building them into the kernel cache. 

Download the latest [release](https://github.com/sakoula/XiaoMi-Air-6200U/releases) from github and unzip the archive. You will find an `AIR_EFI` directory and a `addons` directory. 

Mount the EFI partition of the macOS boot parition on `/Volumes/EFI`:

1. **move** `EFI/CLOVER/kexts/Other/*` from macOS boot parition to `/Library/Extensions/*`
2. run from the console `$ sudo chown -R root:wheel /Library/Extensions/*`
3. run from the console `$ sudo chmod -r 755 /Library/Extensions/*`
4. run from the console `$ sudo kextcache -i /` to rebuild the caches
5. **move** `addons/LiluFriendLite.kext` from the downloaded file to `/Library/Extensions/LiluFriendLite.kext`
6. run from the console `$ sudo chown -R root:wheel /Library/Extensions/*`
7. run from the console `$ sudo chmod -r 755 /Library/Extensions/*`
8. run from the console `$ sudo kextcache -i /` to rebuild the caches

**remember** that `kextcache` needs to be run twice

**Comment by Nxys** [comment]
My error is before the installation, Clover log shows “Kernel Extensions in backtrace: com.apple.driver.AppleIntelSKLGraphicsFramebuffer...”

**Comment by sakoula** [comment]
I assume you are using 10.14.4 . As I said I have not tried it with this version but with 10.4.3 . I will try later today to create a usb stick with 10.14.4 and replicate the problem.

**Comment by Nxys** [comment]
This time I try to download 10.14.3(both 18D42 18D109),then Clover log show:
![微信图片_20190502174627](https://user-images.githubusercontent.com/8848452/57067772-8299f200-6d02-11e9-9a81-98d04761fd7f.jpg)

**Comment by sakoula** [comment]
I will try to replicate later today. Is your laptop the first generation one with the 6200U processor?

**Comment by Nxys** [comment]
Yes, Xiaomi Air 13.3" i5-6200u

**Comment by sakoula** [comment]
Last time I tried it it worked for me. Let me do it again today and will get back to you. The attached EFI is the one you are having on the USB stick right?

**Comment by Nxys** [comment]
That's it.

**Comment by sakoula** [comment]
Hi, I checked it on my laptop and indeed it did not boot for me either. I am not sure with what macOS version I have tested it. The problem appears to be the clover version. I tried with clover 4920 and it is working fine. At some point I will take a closer look and update the guide. For the time being just use the attached EFI folder (copy it over the EFI on the USB stick) and it will work. Let me know how it will go.

[EFI_10.14.4.zip](https://github.com/sakoula/XiaoMi-Air-6200U/files/3146703/EFI_10.14.4.zip)
